### ISSUE TITLE
WARN ignoring uplink error

### ISSUE THREAD

**Issue by pixelchrome** [helium/gateway-rs#46]
I've setup a Dragino LPS8 as a Light-GW. But I can not see that my device (Dragino LGT92) connects to helium properly.

Release v1.0.0-alpha.8

Warning message

```
daemon.info helium_gateway[1710]:  WARN ignoring uplink error: Service(Rpc(Status { code: Unimplemented, message: "unknown service helium.router", metadata: MetadataMap { headers: {"content-type": "application/grpc", "trailer": "Grpc-Status", "trailer": "Grpc-Message", "trailer": "Grpc-Status-Details-Bin"} } })), module: router
```

**Comment by pixelchrome** [comment]
Feedback from @madninja via discord

> I’ll do a release that fixes that today. The settings in alpha.8 are not pointing at the staging router and they should

**Comment by madninja** [comment]
This should be fixed in f9efa3d and part of release `v1.0.0-alpha.9`
